src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/ContextInfos/PackageDeprecationMetadataContextInfo.cs

```diff
@@ -9,12 +9,12 @@ namespace NuGet.VisualStudio.Internal.Contracts
 {
     public sealed class PackageDeprecationMetadataContextInfo
     {
-        public string Message { get; }
+        public string? Message { get; }
```

Comment:
Any service or class versioning considerations...
Planning to fix in 16.10 or 16.9.

Reply:
No this would not require any changes to services.  Its codespaces only as we don't do the deserializer for local scenarios so I was not planning on 16.9 as it is not a critical path, we can discuss further if you would like.